.k8s/production/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-court-data-ui
+  namespace: laa-court-data-ui-production
+spec:
+  replicas: 4
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 100%
+  selector:
+    matchLabels:
+      app: laa-court-data-ui
+  template:
+    metadata:
+      labels:
+        app: laa-court-data-ui
+    spec:
+      containers:
+        - name: laa-court-data-ui-app
+          imagePullPolicy: Always
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-get-paid/laa-court-data-ui:set-me
+          ports:
+            - containerPort: 3000
+          readinessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          env:
+            - name: ENV
+              value: 'production'
+            - name: RACK_ENV
+              value: 'production'
+            - name: RAILS_ENV
+              value: 'production'
+            - name: DOMAIN_URL
+              value: 'https://view-court-data.service.justice.gov.uk'
+            - name: COURT_DATA_ADAPTOR_API_URL
+              value: https://laa-court-data-adaptor-prod.apps.live-1.cloud-platform.service.justice.gov.uk/api/internal/v1
+            - name: RAILS_SERVE_STATIC_FILES
+              value: enabled
+            - name: GA_TRACKING_ID
+              value: 'UA-131160087-14'
```

Comment:
The production and staging namespaces/environments will need their own google analytics tracking code